components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -689,7 +714,7 @@ void AtmosphereOutput::set_diagnostics()
 {
   // Create all diagnostics
   for (const auto& fname : m_fields_names) {
-    if (!m_field_mgr->has_field(fname)) {
+    if (!m_io_field_mgr->has_field(fname)) {
```

Comment:
I thought we agreed on computing diags on the model grid, rather than the IO grid. Shouldn't this be `m_sim_field_mgr`?